docs/PersonalApplications.rst

```diff
@@ -120,64 +104,35 @@ image.
 
 To change the profile banner:
 
-1. Mouse over the top right of the grey banner, an icon will appearn |image7| and a tooltip indicating that the banner should have 130 pixels height.
-
-|image8|
-
-2. Click on the icon |image9|, an ``update`` button appears |image92|. 
-
-3. Click on the Update button, a drive from your hard disk will open.
-
-4. Select the desired profile banner image and double click on it.
-
-5. The profile banner image will be automatically loaded.
-
-|image10|
-
-To reset the profile banner, you just need to hover the right corner
-again and this time, you will have two butons:
-
-|image11|
+1. Mouse over the top right of the grey banner, an edit icon appear.
 
--  ``Update`` button which opens a drive from your hard disk and allows you to change your profile banner.
+    .. note:: the banner should have 130 pixels height.
 
--  ``Reset`` button to restore the default grey banner.
+2. Select the desired profile banner image and double click on it.
 
-When scrolling down on your profile page and if you exceed the cover
-section limit, the profile banner will be replaced by a new navigation
-bar:
+3. The profile banner image will be automatically loaded.
 
-|image13|
+In mobile devices, your profile display is divided on 2 sections: 
+- section 1: About me, , Kudos, Cauris and Work experiences
+- section 2: Welcome to my profile, Contact informations and Badges by domains
 
-This new navigation bar contains:
+To switch from both sections, you can use the swipe icon.
 
--  Your avatar in a cercle form.
-
--  Your display name and your chat status.
-
--  The edit button |image14| allowing you to edit your profile.
-
--  Your applications navigation bar.
-
--  The *More* button |image15| which appears when applications display
-   exceed the navigation bar limit.
-
-   |image16|
-
-In mobile devices, when scrolling down your profile page, the
-applications navigation bar is scrollable left and right.
-
-|image17|
+|PRFL-3|
 
 .. _About-me:
 
 About Me
 -----------
 
-A brief introduction of yourself with 1500 characters in maximum,
+A brief introduction of yourself with 2000 characters in maximum,
```

Comment:
```suggestion
A brief introduction of yourself (in less than 2000 characters),
```

functionnally, it would be a better UX it it was expressed in words.